php/lemon/config.yaml

```
@@ -0,0 +1,3 @@
+framework:
+  github: Lemon-Framework/Lemon
+  version: 3.10.1
```

we need to force as string, cause `3.10` will show `3.1` in results

```suggestion
  version: '3.10'
```